actors/util/adt/map.go

```diff
@@ -17,6 +18,15 @@ import (
 // may differ, in which case we can expose it for configuration.
 const hamtBitwidth = 5
 
+// HamtOptions specifies all the options used to construct filecoin HAMTs.
+var HamtOptions = []hamt.Option{
+	hamt.UseTreeBitWidth(hamtBitwidth),
+	hamt.UseHashFunction(func(input []byte) []byte {
+		res := sha256.Sum256(input)
```

Comment:
@whyrusleeping said sha256 is faster on his AMD machine than murmur3 was anyways.